Documentation.md

```diff
@@ -16,6 +16,10 @@ Click [here](Architecture.md) to view a quick architecture overview of how the
 Amazon Connect Streams API works.
 
 ## Getting Started
+
+### Upgrading to the OmniChannel CCP (AKA CCPv2)?
+If you are migrating to the new CCP, we encourage you to upgrade to the latest version of this repository. You should also upgrade to [the latest version of RTC-JS](https://github.com/aws/connect-rtc-js) as well, if you are using it. For a complete migration guide to the new CCP, and to fully understand the differences when using Streams with the new CCP, please see this post: https://docs.aws.amazon.com/connect/latest/adminguide/upgrade-to-latest-ccp.html
```

Comment:
If you are migrating to the latest version of the CCP (url ends with /ccp-v2)

Reply:
Since these customers will be Streams API customers by definition (right?), we should send them the sub-article: https://docs.aws.amazon.com/connect/latest/adminguide/upgrade-ccp-streams-api.html